### DISCUSSION TITLE
Is it possible to configure default properties according to button size?

### DISCUSSION THREAD

**Discussion by borachoidev** [mantinedev/mantine#6029] [Q&A]
I would like to customize button styles based on their sizes. Specifically, if the button size is set to "sm", the font weight should be 400; for a size of "md", the font weight should be 500, among other specifications. Is it possible to set default properties separately for each button size?

Thank you for your assistance.

**Comment by rtivital** [comment]
https://mantine.dev/styles/variants-sizes/#sizes-with-data-size-attribute
